src/utils/validateIdentifierInputs.ts

```diff
@@ -19,19 +19,20 @@ export const validateIdentifierInputs = (
   identifierInputs: IdentifierInput[]
 ): boolean => {
   for (const { value, validationRegex, textInputType } of identifierInputs) {
-    if (textInputType === "SINGLE_CHOICE" && !value) {
+    const trimmedValue = value.trim();
```

Comment:
notice the first 2 `if` conditions are checking if `value` exist. since there's a possibility that `value` is undefined, do you think we need to passed the trimmed value in here?

Reply:
didn't catch this while changing all the values, thanks, updated!

Reply:
no problem! do you think it will be better to move the value.trim() to after these checks as well? since it's not used for the 3 conditions!

Reply:
moved it! i thought about doing that but I left it there because I thought it might be used somewhere in the checks in the future